scripts/get-release-artifact-and-commit-sha.js

```diff
@@ -19,7 +19,7 @@ async function main() {
     'stable',
   ].join('/');
   const binariesMatcher =
-    /^coveo[_-]{1}(?<_version>v?\d+\.\d+\.\d+(-\d+)?)[_-]{1}(?<_commitSHA>\w{7})[_-]{1}(?<longExt>.*\.(exe|deb|pkg))$/;
+    /^coveo[_-]{1}(?<_version>v?\d+\.\d+\.\d+(-\d+)?)[_.-]{1}(?<_commitSHA>\w{7})[_-]{0,1}(\d+_)?(?<longExt>.*\.(exe|deb|pkg))$/;
```

Comment:
could this happen **`(\d+[_-]{1})`** ?

Reply:
I did not see it yet. If it happens, well, we'll modify it again gee